frontend/components/Domain/Recipe/RecipeIngredients.vue

```diff
@@ -29,6 +29,7 @@
         <v-list-item
           density="compact"
           @click.stop="toggleChecked(index)"
+          class="pa-0"
```

Comment:
event triggers should always be at the bottom
```suggestion
          class="pa-0"
          @click.stop="toggleChecked(index)"
```